package.json

```diff
@@ -11,14 +11,12 @@
   "author": "Quami Killy",
   "license": "MIT",
   "devDependencies": {
-    "@babel/core": "^7.18.2",
-    "@babel/preset-env": "^7.18.2",
     "babel-eslint": "^10.1.0",
-    "babel-loader": "^8.2.5",
     "css-loader": "^6.7.1",
     "eslint": "^7.32.0",
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-plugin-import": "^2.26.0",
+    "eslint-webpack-plugin": "^3.1.1",
```

Comment:
[OPTIONAL] One way you can troubleshoot the linters issue is to re-install all your ESlint and Stylelint packages by doing the following:

1. In your `package.json` delete the Eslint and Stylelint dependencies manually.
![image](https://user-images.githubusercontent.com/38088818/173058845-2ba08921-00c7-4c36-8882-e13003c01cfe.png)

2. Then, delete the `package-lock.json` file and `node_modules` directory.
3. Then, run `npm install` to re-install the `package-lock.json` and `node_modules` without Eslint and Stylelint dependencies.

4. Then, go to the linter checks details on your pull request.
![image](https://user-images.githubusercontent.com/38088818/173059521-8cdd16ee-dd17-4d9b-8b0b-041fcd1768ff.png)

- Copy the Stylelint command from Github and use it to install Stylelint.
![image](https://user-images.githubusercontent.com/38088818/173060097-13cbfde5-84b6-4326-9c2e-fadb9b295470.png)

- Similarly, Copy the Eslint command from Github and use it to install Eslint as well.
![image](https://user-images.githubusercontent.com/38088818/173062470-97df76bc-d75c-4b7a-98c4-3b47ab47890e.png)

5. Commit and push changes.

Reply:
@KDlamini  this is very helpful thanks for the solution